src/test/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFsIntegrationTest.java

```diff
@@ -588,7 +588,7 @@ public void testCombineReblockedGVCFs() {
         Assert.assertFalse(g0.hasPL());
         Assert.assertFalse(g0.hasGQ());
         Assert.assertFalse(g0.hasExtendedAttribute(VCFConstants.DEPTH_KEY));
-        Assert.assertTrue(g1.isHomRef());
+        Assert.assertTrue(g1.isNoCall());
```

Comment:
In a lot of these tests I was explicitly testing the behavior of the GQ0 hom-ref, but that's exactly what we're trying to change here